Dockerfile

```
@@ -0,0 +1,24 @@
+FROM alpine:3.10 AS builder
+LABEL builder=true
+
+ENV CGO_ENABLED=0
+ENV GOPATH /go
+
+RUN apk add --update -t build-deps go git mercurial libc-dev gcc libgcc
```

```suggestion
RUN apk add --no-cache -t build-deps go git mercurial libc-dev gcc libgcc
```

You don't update base image packages unless it is needed for a reason.